chatcommands.py

```diff
@@ -1092,6 +1092,47 @@ def report(msg, urls):
         return os.linesep.join(output)
 
 
+# noinspection PyIncorrectDocstring,PyUnusedLocal
+@command(str, whole_msg=True, privileged=True, aliases=['scan', 'test-p'])
+def checkpost(msg, url):  # FIXME: Currently does not support batch report
+    """
+    Force Smokey to scan a post even if it has no recent activity
+    :param msg:
+    :param url:
+    :return I don't know:
+    """
+
+    crn, wait = can_report_now(msg.owner.id, msg._client.host)
```

Comment:
`can_report_now` shouldn't be called as this command doesn't call the API, it doesn't use API quota and won't be rate limited.

Reply:
This does call the API, albeit indirectly.